cwms-data-api/src/test/java/cwms/cda/api/MeasurementControllerTestIT.java

```diff
@@ -0,0 +1,630 @@
+                .body("[0].supplemental-streamflow-measurement.surface-velocity", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getSurfaceVelocity().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.max-velocity", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getMaxVelocity().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.effective-flow-area", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getEffectiveFlowArea().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.cross-sectional-area", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getCrossSectionalArea().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.mean-gage", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getMeanGage().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.top-width", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getTopWidth().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.main-channel-area", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getMainChannelArea().floatValue()))
+                .body("[0].supplemental-streamflow-measurement.overbank-area", equalTo(updatedMeasurement.getSupplementalStreamflowMeasurement().getOverbankArea().floatValue()))
+                .body("[0].usgs-measurement.remarks", equalTo(updatedMeasurement.getUsgsMeasurement().getRemarks()))
+                .body("[0].usgs-measurement.current-rating", equalTo(updatedMeasurement.getUsgsMeasurement().getCurrentRating()))
+                .body("[0].usgs-measurement.control-condition", equalTo(updatedMeasurement.getUsgsMeasurement().getControlCondition()))
+                .body("[0].usgs-measurement.flow-adjustment", equalTo(updatedMeasurement.getUsgsMeasurement().getFlowAdjustment()))
+                .body("[0].usgs-measurement.shift-used", equalTo(updatedMeasurement.getUsgsMeasurement().getShiftUsed().floatValue()))
+                .body("[0].usgs-measurement.percent-difference", equalTo(updatedMeasurement.getUsgsMeasurement().getPercentDifference().floatValue()))
+                .body("[0].usgs-measurement.delta-height", equalTo(updatedMeasurement.getUsgsMeasurement().getDeltaHeight().floatValue()))
+                .body("[0].usgs-measurement.delta-time", equalTo(updatedMeasurement.getUsgsMeasurement().getDeltaTime().floatValue()))
+                .body("[0].usgs-measurement.air-temp", equalTo(updatedMeasurement.getUsgsMeasurement().getAirTemp().floatValue()))
+                .body("[0].usgs-measurement.water-temp", equalTo(updatedMeasurement.getUsgsMeasurement().getWaterTemp().floatValue()));
+
+        // Delete the Measurement
```

Comment:
Consider adding measurement deletion to the afterall cleanup routine, just in case the test fails before cleanup can occur within the test.

Reply:
Good idea. Added measurement deletion call to afterall.